Formula/c/color-code.rb

```diff
@@ -25,24 +24,21 @@ class ColorCode < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "49bf9128a81486c66e92b0382a448a970b20c32a01d296aaeeca8d1a9c4995c2"
   end
 
+  depends_on "cmake" => :build
   depends_on "qt@5"
 
   fails_with gcc: "5"
 
   def install
-    qt5 = Formula["qt@5"].opt_prefix
-    system "#{qt5}/bin/qmake"
-    system "make"
-
-    if OS.mac?
-      prefix.install "ColorCode.app"
-      bin.write_exec_script "#{prefix}/ColorCode.app/Contents/MacOS/colorcode"
-    else
-      bin.install "colorcode"
-    end
+    system "cmake", "-S", "src", "-B", "build_cmake", *std_cmake_args
+    system "cmake", "--build", "build_cmake"
+    system "cmake", "--install", "build_cmake"
```

Comment:
do we still need this step?
```suggestion
```

Reply:
Good point. Forgot to remove that when debugging as we no longer need it.